src/python_ml_project_template/datasets/flowbot.py

```diff
@@ -3,17 +3,26 @@
 import lightning as L
```

Comment:
note in general - the directory src/python_ml_project_template should be renamed to 
```
src/open_anything_diffusion
```